src/main/kotlin/no/nav/syfo/sykmelding/SykmeldingService.kt

```diff
@@ -26,6 +26,11 @@ class SykmeldingService(
     private val sykmeldingStatusRedisService: SykmeldingStatusRedisService,
     private val pdlPersonService: PdlPersonService
 ) {
+    suspend fun hentSykmelding(token: String, sykmeldingid: String): SykmeldingDTO {
+        return syfosmregisterSykmeldingClient.getSykmelding(token = token, sykmeldingid = sykmeldingid)
+            .run(this::getSykmeldingWithLatestStatus)
```

Comment:
Gjør run det samme som map, bare for et element? Har ikke brukt run noe særlig selv :)

Reply:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/run.html